Tests.Integration.Presentation.Web/Contract/V2/ItContractsApiV2Test.cs

```diff
@@ -367,6 +367,136 @@ public async Task Cannot_POST_With_Duplicate_Name_In_Same_Org()
             Assert.Equal(HttpStatusCode.Conflict, duplicateResponse.StatusCode);
         }
 
+        [Fact]
+        public async Task Can_POST_With_Parent()
+        {
+            //Arrange
+            var (token, user, organization) = await CreatePrerequisitesAsync();
+            var parent = await ItContractHelper.CreateContract(CreateName(), organization.Id);
+
+            var requestDto = new CreateNewContractRequestDTO()
+            {
+                OrganizationUuid = organization.Uuid,
+                Name = CreateName(),
+                ParentContractUuid = parent.Uuid
+            };
+
+            //Act
+            var contractDTO = await ItContractV2Helper.PostContractAsync(token, requestDto);
+
+            //Assert
+            Assert.Equal(parent.Name, contractDTO.ParentContract.Name);
+            Assert.Equal(parent.Uuid, contractDTO.ParentContract.Uuid);
+        }
+
+        [Fact]
+        public async Task Cannot_POST_With_Parent_From_Other_Organization()
+        {
+            //Arrange
+            var (token, user, organization1) = await CreatePrerequisitesAsync();
```

Comment:
Hvis brugeren havde været i en anden organization så havde den her test været grøn alligevel for din forretningslogik tjekker ikke på at de er i samme org - kun at nuværende bruger kan hente den ud